### PR TITLE
feat(web): add visual table editor dialog

### PR DESCRIPTION
**Pull request by yanglbme**
## Summary

Replace the simple insert-form dialog with a full visual table editor:

- **Create/edit dialog**: row/column steppers, per-column alignment cycling, cell keyboard navigation (Enter moves down and appends rows), delete table/row/column actions. Stepper caps gate new growth only — existing oversized tables are never truncated.
- **Multiple entry points**: insert dropdown, slash command (`/table`), command palette, and a position-aware editor context menu item (`posAtCoords`, so right-clicking a table works regardless of the caret position).
- **Preview click-to-edit**: single click on a preview table (with `cursor: pointer`, excluding blockquote-nested tables) opens the editor. Preview tables are mapped to source ranges positionally via a new marked-parity scanner in `@md/shared` (`findAllMarkdownTables`): header/delimiter cell-count equality, fence/blockquote/indented-code exclusion, and pipe-less body continuation lines all mirror `marked@18` behavior.
- **Robust writes**: editing preserves the original indentation (list-nested tables stay nested); if the document changed while the dialog is open, the target table is re-located by content (`relocateMarkdownTable`) instead of writing at stale offsets; deleting a table swallows the surplus blank line.
- **Incidental fix**: wrap imperative `ResizablePanel.resize()` calls in a deferred `try/catch` helper to silence reka-ui "Assertion failed!" console errors on mount/HMR.

The old `InsertFormDialog` is removed; `cursor: pointer` lives in unscoped preview CSS that is not part of the copy pipeline, so copying to WeChat is unaffected.

## Type of Change

- [x] New feature
- [x] Bug fix (reka-ui resize assertion)

## Test Procedure

- [x] 41 unit tests for `packages/shared/src/utils/table.ts` pass (`vitest`), covering marked-parity scanning, parsing, building, locating, and relocation edge cases
- [x] `pnpm run lint` (eslint --fix) clean
- [x] `pnpm web build` passes including full `vue-tsc --build --force` type check
- [x] Manually verified in dev server: create via dropdown/slash command, edit via context menu and preview click, alignment cycling, row/column add/remove, delete, and copy-to-WeChat output unchanged

## Pre-flight Checklist

- [x] Code follows project conventions (ESLint clean, English comments)
- [x] i18n strings added for zh-CN / zh-TW / en-US / ja-JP
- [x] No secrets or unrelated changes included

### CODE REVIEW

File: apps/web/src/stores/ui.ts [hunk 1]
```diff
@@ -50,8 +50,15 @@ export const useUIStore = defineStore(`ui`, () => {
...
-  const toggleShowInsertFormDialog = useToggle(isShowInsertFormDialog)
+  const isShowTableEditDialog = ref(false)
+
+  function openTableEditDialog() {
+    isShowTableEditDialog.value = true
+  }
+
+  function closeTableEditDialog() {
+    isShowTableEditDialog.value = false
+  }
```
[the remark this answers is not in the document]
Comment (yanglbme) [reply apps/web/src/stores/ui.ts:61]: Fixed in 02c7a5c8 — the shared storeLabels key is now isShowTableEditDialog, matching the renamed UI store state and the i18n editorState.storeLabels labels.
